### PR TITLE
Liberty: Custom Unit for Lord Maddock

### PR DESCRIPTION
**Pull request by Dalas121**
Elensefar's Lord Maddock makes an appearance in Liberty's scenario 4 "Unlawful Orders", where he's represented by the General unit.  As a somewhat significant character, I say he deserves his own unit, which I've added in this PR.

Image based on the Cavalier, with added animations.  If my contributions need a license, CC-BY-SA 4.0 is fine.

Also, are animations of this low-ish quality acceptable for mainline units?  I could put together some similar attack animations for the regular Cavalier, who still doesn't have any.

### CODE REVIEW

**Comment by knyghtmare** [comment]
A whole new unit...just for one scenario?

**Comment by Dalas121** [comment]
Hah. I made the baseframe for a cutscene in TDG, and figured I might as well keep it consistent with the Liberty version, so I added animations and made this PR.

**Comment by knyghtmare** [comment]
Hmm, Ashievere from HttT also has a custom unit (despite only being present on map in S24) so I guess Maddock getting his own unit for a single scenario is okay.

File: data/campaigns/Liberty/units/Maddock.cfg [hunk 1]
```diff
@@ -0,0 +1,74 @@
+#textdomain wesnoth-l
+[unit_type]
+    id=Lord of Elensefar
+    name= _ "Lord of Elensefar"
+    race=human
+    profile="portraits/maddock.webp"
+    description=_"The grand city-state of Elensefar is the wealthiest human settlement on the great continent, and second in size only to the sprawling Dan-Tonk. Elensefar is at times a province of Wesnoth, at times an independent country, and at times in a treaty federation.
```
Comment (nemaara) [inline data/campaigns/Liberty/units/Maddock.cfg:7]:
```suggestion
    description=_"The grand city-state of Elensefar is the wealthiest human settlement on the Great Continent, commanding a reputation and power that belies its relatively small population. Unlike the more war-minded inland nobles, Elensefar’s governors master the art of commerce and the skill of sailing and are known to be particularly agreeable. Yet behind that merchant charm lies a fierce independence, a trait that sometimes puts them at odds with the aristocrats of other regions.
```

Comment (nemaara) [reply data/campaigns/Liberty/units/Maddock.cfg:7]: This change would also delete the line below.

File: data/campaigns/Liberty/images/units/maddock/maddock-armed.png
Comment (nemaara) [inline data/campaigns/Liberty/images/units/maddock/maddock-armed.png:1]: How would you feel about adding some kind of crest? The top of the helmet looks kind of plain?

Comment (Dalas121) [reply data/campaigns/Liberty/images/units/maddock/maddock-armed.png:1]: Maddock's portrait doesn't have any ornamentation on top of his helmet (or his chainmail), so I think it's probably ok?

Comment (nemaara) [reply data/campaigns/Liberty/images/units/maddock/maddock-armed.png:1]: okaydokay

**Review by nemaara — APPROVED** [review]

**Comment by Dalas121** [comment]
How do I handle merging this in with the copyrights.csv conflict?

**Comment by soliton-** [comment]
> How do I handle merging this in with the copyrights.csv conflict?

The conflict needs to be resolved. If you want someone else can probably do that for you.

**Comment by Dalas121** [comment]
For now I've only been using the Github web UI, and it looks like the web UI's resolve conflicts button can't handle this CSV.  Could you resolve the conflict for this PR, and I'll figure out how to do this myself for future ones?

**Comment by soliton-** [comment]
Well, this too messed up. Even after multiple corrections I cannot tell what the actual additions are. Please post those then I can push a new copyrights.csv.

**Comment by Dalas121** [comment]
Thanks!  Merged.

File: data/campaigns/Liberty/units/Maddock.cfg [hunk 2]
```diff
@@ -0,0 +1,72 @@
+#textdomain wesnoth-l
+[unit_type]
+    id=Lord of Elensefar
+    name= _ "Lord of Elensefar"
+    race=human
+    profile="portraits/maddock.webp"
+    description=_"The grand city-state of Elensefar is the wealthiest human settlement on the Great Continent, commanding a reputation and power that belies its relatively small population. Unlike the more war-minded inland nobles, Elensefar’s governors master the art of commerce and the skill of sailing, and are known to be particularly agreeable. Yet behind that merchant charm lies a fierce independence, a trait that sometimes puts them at odds with the aristocrats of other regions."
```
Comment (Wedge009) [inline data/campaigns/Liberty/units/Maddock.cfg:7]: I suppose it's not a big deal if it's left as-is, but I wonder if 'belligerent' might be a better replacement for 'war-minded'.

**Pull request merged**